.github/workflows/maintainance_bot.yaml

```diff
@@ -4,26 +4,23 @@ on:
 
 jobs:
   labeler:
-    name: "Assign Labels"
+    name: Assign labels and milestone
     runs-on: ubuntu-latest
+    env:
+      MILESTONE_NUMBER: 18
     steps:
       - uses: actions/labeler@main
         with:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
-
-  milestone:
-    name: "Assign Milestone"
-    runs-on: ubuntu-latest
-    steps:
-      - name: Assign Milestone
-        if:  |
-          contains(github.event.pull_request.title, 'WIP') == false &&
-          contains(github.event.pull_request.title, 'merge') == false &&
-          github.event.pull_request.draft == false
-        env:
-          MILESTONE_NUMBER: 18
+      - name: Assign milestone
+        if: |
+          ! github.event.pull_request.milestone &&
+          ! contains(github.event.pull_request.labels.*.name, 'merge') &&
+          ! contains(github.event.pull_request.labels.*.name, 'status/WIP') &&
+          ! contains(github.event.pull_request.title, 'WIP') &&
+          ! github.event.pull_request.draft
```

Comment:
do we have label `merge`? Anyway, I also saw negation `!` operator in gha docs, but it didn't work for me. I will try this out now.

Reply:
hm, maybe space was the trick.  `!` works fine, I tried it and it works perfectly. Thanks a bunch @nsoranzo

Reply:
> do we have label `merge`? Anyway, I also saw negation `!` operator in gha docs, but it didn't work for me. I will try this out now.

Yes: https://github.com/galaxyproject/galaxy/labels/merge . It's used for backport PRs, so that these are not picked up by the script that generate the release notes. Actually the labeler should also skip them.